mlflow/pyfunc/backend.py

```diff
@@ -68,7 +68,7 @@ def __init__(
         self,
         config,
         workers=1,
-        env_manager=_EnvManager.VIRTUALENV,
+        env_manager=None,
```

Comment:
Can we remove `None` default? Then we can avoid incorrect assumption of default values.